apycula/gowin_pack.py

```diff
@@ -108,7 +108,7 @@ def header_footer(db, bs):
     CRC_check and security_bit_enable set
     """
     bs = np.fliplr(bs)
-    bs=np.packbits(bs, axis=1)
+    bs=np.packbits(bs)
     # configuration data checksum is computed on all
     # data in 16bit format
     bb = np.array(bs.flat)
```

Comment:
Am I correct in thinking that this `flat` is unneeded now, because `packbits` without axis arguments already flattens the array?